### DISCUSSION TITLE
Medium icon size on taskbar?

### DISCUSSION THREAD

**Discussion by marc6h** [valinet/ExplorerPatcher#2616] [Ideas]
Hi, in the Windows 10 taskbar mode, there are 2 options for the icon size: Large and Small. The difference is very big. Would it be possible to add a Medium option for this? I know it wasn't in the original Windows 10, but would be interested in this anyway if possible. Thanks.

**Comment by pyrates999** [comment]
That would be something that MS would have to add code for.
